### DISCUSSION TITLE
How to use language selector?

### DISCUSSION THREAD

**Discussion by Toxumuharu** [binokochumolvarghese/lightbi-hugo#29] [Q&A]
Hi @binokochumolvarghese, thank you for distributing this nice theme.
I've been struggling to have language switcher in my environment.

Is it the same way to have the switcher following below?
- https://gohugo.io/content-management/multilingual/

Thanks ahead.

**Comment by binokochumolvarghese** [comment]
Hi, I am happy to hear you liked the theme.

Yes you can follow the same way and you can checkout the [Lightbi Wiki page](https://github.com/binokochumolvarghese/lightbi-hugo/wiki/General-Features#multilingual) if you want.

**Comment by Toxumuharu** [comment]
Thank you @binokochumolvarghese , it looks like finally I made it!  - https://toxumuharu.github.io/TomoharuSecretTravelBag/en/
Getting another problem, in the top page somehow the description isn't shown. When I tried it with single language, it appeares...

I would solve this issue and how can I improve my preference?
Thanks ahead.
